server-v2/api/studio/studio.go

```diff
@@ -38,6 +54,25 @@ func main() {
 
 	// global middleware
 	server.Use(auth.AuthMiddlewareWithCtx(svcCtx))
+	server.Use(rest.ToMiddleware(middleware.ReserveRequest(middleware.ReserveRequestConfig{
+		Skipper: func(r *http.Request) bool {
+			if strings.HasPrefix(r.URL.Path, "/api/files") {
+				return false
+			}
+			if strings.HasPrefix(r.URL.Path, "/api/import-tasks/download") {
```

Comment:
There are several places with this logic,  can you encapsulated into a function?